satellite/gc/piecetracker.go

```diff
@@ -20,7 +20,10 @@ type PieceTracker struct {
 	log          *zap.Logger
 	config       Config
 	creationDate time.Time
-	pieceCounts  map[storj.NodeID]int
+	// TODO: does it make sense to name this type?
+	//  (If so, where to put it that doesn't create import cycle?)
+	// TODO: should we use int or int64 consistently for piece count (db type is int64)?
```

Comment:
let's use int64

Reply:
as discussed we'll do this in another PR